src/test/scala/cromwell/webservice/CromwellApiServiceSpec.scala

```diff
@@ -653,15 +670,15 @@ class CromwellApiServiceSpec extends FlatSpec with CromwellApiService with Scala
 
   "Cromwell query API" should "return 400 for a bad query" in {
     Get(s"/workflows/$version/query?BadKey=foo") ~>
-    queryRoute ~>
-    check {
-      assertResult(StatusCodes.BadRequest) {
-        status
+      queryRoute ~>
+      check {
+        assertResult(StatusCodes.BadRequest) {
+          status
```

Comment:
It does test that the plumbing between `CromwellApiService` and `CromwellApiHandler` is sound, but really no more than that.  The structure of these `CromwellApiServiceSpec` tests does feel kind of wrong.